examples/1d/plot_filters.py

```diff
@@ -1,164 +1,143 @@
+# -*- coding: utf-8 -*-
+"""plot_filters.ipynb
+
+Automatically generated by Colaboratory.
```

Comment:
Could you restore the original formatting (i.e., none of this auto-generated stuff from Colab)?

Reply:
Sure, I did not notice Colab does this, sry.